src/v/cloud_storage/probe.cc

```diff
@@ -29,126 +29,147 @@ remote_probe::remote_probe(
+             sm::description("Number of times backoff was applied during "
+                             "manifest download")),
+           sm::make_counter(
+             "successful_uploads",
+             [this] { return get_successful_uploads(); },
+             sm::description("Number of completed log-segment uploads")),
+           sm::make_counter(
+             "successful_downloads",
+             [this] { return get_successful_downloads(); },
+             sm::description("Number of completed log-segment downloads")),
+           sm::make_counter(
+             "failed_uploads",
+             [this] { return get_failed_uploads(); },
+             sm::description("Number of failed log-segment uploads")),
+           sm::make_counter(
+             "failed_downloads",
+             [this] { return get_failed_downloads(); },
+             sm::description("Number of failed log-segment downloads")),
+           sm::make_counter(
+             "failed_manifest_uploads",
+             [this] { return get_failed_manifest_uploads(); },
+             sm::description("Number of failed manifest uploads")),
+           sm::make_counter(
+             "failed_manifest_downloads",
+             [this] { return get_failed_manifest_downloads(); },
+             sm::description("Number of failed manifest downloads")),
+           sm::make_counter(
+             "upload_backoff",
+             [this] { return get_upload_backoffs(); },
+             sm::description("Number of times backoff was applied during "
+                             "log-segment uploads")),
+           sm::make_counter(
+             "download_backoff",
+             [this] { return get_download_backoffs(); },
+             sm::description("Number of times backoff  was applied during "
+                             "log-segment downloads")),
+           sm::make_counter(
+             "bytes_sent",
+             [this] { return _cnt_bytes_sent; },
+             sm::description("Number of bytes sent to cloud storage")),
+           sm::make_counter(
+             "bytes_received",
+             [this] { return _cnt_bytes_received; },
+             sm::description("Number of bytes received from cloud storage")),
+           sm::make_counter(
+             "index_uploads",
+             [this] { return _cnt_index_uploads; },
+             sm::description("Number of segment indices uploaded")),
+           sm::make_counter(
+             "index_downloads",
+             [this] { return _cnt_index_downloads; },
+             sm::description("Number of segment indices downloaded")),
+           sm::make_counter(
+             "failed_index_uploads",
+             [this] { return _cnt_failed_index_uploads; },
+             sm::description("Number of failed segment index uploads")),
+           sm::make_counter(
+             "failed_index_downloads",
+             [this] { return _cnt_failed_index_downloads; },
+             sm::description("Number of failed segment index downloads"))});
```

Comment:
nit: maybe using a comma after the last element will preserve the formatting

Reply:
fixed, seems to have made formatting correct like before